### PR TITLE
Improve readability in some code and change way that PhulpFile works

### PR DESCRIPTION
**Pull request by oliveiramiguel**
Remove some "else" to improve readability e maintance.
As noticed on #10, I liked that way to code the taks (without PhulpFule class).

### CODE REVIEW

File: src/PipeInterface.php [hunk 1]
```diff
@@ -7,5 +7,6 @@
     /**
      * @param Source $source
      */
+
```
Comment (reisraff) [inline src/PipeInterface.php]: why?

**Comment by reisraff** [comment]
@oliveiramiguel can you update the DOCUMENTATION.md? After this squash the commits.

I liked this approach.

File: src/PipeInterface.php (hunk 1 above)
Comment (oliveiramiguel) [inline src/PipeInterface.php]: I little mistake :(

**Comment by oliveiramiguel** [comment]
@reisraff DOCUMENTATION.md, It already has been updated, Do you missing something?

**Comment by reisraff** [comment]
@oliveiramiguel Ah okay sorry, so just squash the commits please, I'll merge it.

**Pull request merged**